packages/scripts/src/commands/app-data/postProcess/assets.ts

```diff
@@ -238,66 +255,58 @@ export class AssetsPostProcessor {
+        [baseFolder, ...nestedPaths] = nestedPath.split("/");
+        nestedPath = nestedPaths.join("/");
+      }
+
+      if (baseFolder === globalFolder || isCountryLanguageCode(baseFolder)) {
+        entries.complete[nestedPath] ??= {} as any;
+        entries.complete[nestedPath].themeVariations ??= {};
+        entries.complete[nestedPath].themeVariations[themeName] ??= {};
+        entries.complete[nestedPath].themeVariations[themeName][baseFolder] = assetEntry;
       } else {
-        entries.translations[baseFolder] ??= {};
-        entries.translations[baseFolder][nestedPath] = assetEntry;
+        console.log(
+          `Folder naming error: The folder "${baseFolder}" is not named with a valid language code`
+        );
       }
     });
 
-    mergeAssetOverrides("translations");
-    mergeAssetOverrides("themeVariations");
+    // Check for assets which have no default version, and move them to "orphaned"
+    Object.entries(entries.complete).forEach(([assetPath, assetEntry]) => {
+      if (!assetEntry.themeVariations.default?.hasOwnProperty("global")) {
+        entries.orphaned[assetPath] = assetEntry;
+        delete entries.complete.assetPath;
+      }
+    });
     return entries;
-
-    /** Update global asset entries to include language or theme overrides */
-    function mergeAssetOverrides(type: "translations" | "themeVariations") {
-      // process assets by language, copying a reference for any language assets to corresponding
-      // global. Track as missing if global does not exist
-      Object.entries(entries[type]).forEach(([baseName, langHashmap]) => {
-        Object.entries(langHashmap).forEach(([nestedPath, entry]) => {
-          if (entries.global.hasOwnProperty(nestedPath)) {
-            entries.global[nestedPath][type] ??= {};
-            entries.global[nestedPath][type][baseName] = entry;
-          } else {
-            // add metadata to track where missing asset came from
-            entry[`_source`] = `${type} - ${baseName}`;
-            entries.missing[nestedPath] = entry;
-          }
-        });
-      });
-    }
   }
 
   /** Strip additional fields from contents entry to provide cleaner asset entry */
-  private contentsToAssetEntry(entry: IContentsEntry): IAssetEntry {
+  private contentsToAssetEntry(entry: IContentsEntry): IContentsEntryMinimal {
     const { md5Checksum, size_kb } = entry;
     return { size_kb, md5Checksum };
   }
 }
 
 interface IAssetEntriesByType {
-  /** Assets listed in global folder with reference to language or theme overrides */
-  global: IAssetEntryHashmap;
-  translations: { [languageCode: string]: IAssetEntryHashmap };
-  themeVariations: { [themeName: string]: IAssetEntryHashmap };
-  /** Assets that appear in translation folders but have no corresponding globals */
-  missing: IAssetEntryHashmap;
+  /** Assets that have a global in the default theme, including their respective overrides */
+  complete: IAssetEntryHashmap;
```

Comment:
nit(non-blocking)
I might suggest distinguish `tracked` / `untracked` instead of `complete` / `orphaned` as slightly clear naming, but not a strong opinion